simexpal/util.py

```diff
@@ -97,6 +97,19 @@ def _validate_dict(dictionary, source, schema_path):
 
 				print("simexpal: Validation error in {} at {}:\n{}\n{}".format(
 					source, err_source, err.instance, err.message), file=sys.stderr, end="\n\n")
+
+				# The error comes from subschemas in anyOf, oneOf or allOf.
+				if err.context:
```

Comment:
`context` is only set for anyOf, oneOf and allOf? Did you test that is does not give unreasonable output in other cases?

Reply:
> `context` is only set for anyOf, oneOf and allOf? 

Yes, that should be the case. The following text comes from the [official documentation](https://python-jsonschema.readthedocs.io/en/stable/errors/): _With validators like anyOf, the ``ValidationError.context`` attribute can be used to see the sub-errors which caused the failure._

> Did you test that is does not give unreasonable output in other cases?

Yes, I did some testing and in other cases the ``context`` attribute is just an empty list ``[]``.